src/main/java/frc/robot/subsystems/Chassis.java

```diff
@@ -77,41 +63,18 @@ private void setVelocityMode(CANSparkMax sparky) {
         final double kFF = 0.000015;
         final double kMaxOutput = 1;
         final double kMinOutput = -1;
+        final double rr = 0.1;
```

Comment:
This seems like a very slow ramp - what's the motivation?

Reply:
it is a placeholder for now in case the robot is tippy with the climber mounted

Reply:
Even with the top heavy 2019 climber, we ran a 0.7 ramp rate.